common/darwin/Classes/FlutterWebRTCPlugin.m

```diff
@@ -88,13 +88,6 @@ - (instancetype)initWithChannel:(FlutterMethodChannel *)channel
     self.localTracks = [NSMutableDictionary new];
     self.renders = [NSMutableDictionary new];
     self.videoCapturerStopHandlers = [NSMutableDictionary new];
-#if TARGET_OS_IPHONE
-    _preferredInput = AVAudioSessionPortHeadphones;
-    _speakerOn = NO;
-    [AudioUtils setSpeakerphoneOn:_speakerOn];
-    AVAudioSession *session = [AVAudioSession sharedInstance];
-    [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(didSessionRouteChange:) name:AVAudioSessionRouteChangeNotification object:session];
```

Comment:
I think that at last this part must be left here, because technically we could want to get this notification event without active peer connection.